deployments/types/rococo-wococo.json

```diff
@@ -0,0 +1,52 @@
+{
+	"--1": "Rococo Types",
+	"RococoBalance": "u128",
+	"RococoBlockHash": "H256",
+	"RococoBlockNumber": "u32",
+	"RococoHeader": {
+		"parent_Hash": "RococoBlockHash",
+		"number": "Compact<RococoBlockNumber>",
+		"state_root": "RococoBlockHash",
+		"extrinsics_root": "RococoBlockHash",
+		"digest": "RococoDigest"
+	},
+	"RococoDigest": {
+		"logs": "Vec<RococoDigestItem>"
+	},
+	"RococoDigestItem": {
+		"_enum": {
+			"Other": "Vec<u8>",
+			"AuthoritiesChange": "Vec<AuthorityId>",
+			"ChangesTrieRoot": "RococoBlockHash",
+			"SealV0": "SealV0",
+			"Consensus": "Consensus",
+			"Seal": "Seal",
+			"PreRuntime": "PreRuntime"
+		}
+	},
+	"--2": "Wococo Types",
+	"WococoBalance": "u128",
```

Comment:
Isn't Rococo and Wococo always going to be the same? I feel this is a bit duplicated.